GDJS/Runtime/runtimeobject.js

```diff
@@ -34,27 +41,64 @@
  * @param {ObjectData} objectData The initial properties of the object.
  */
 gdjs.RuntimeObject = function(runtimeScene, objectData) {
+    /** @type {string} */
     this.name = objectData.name || "";
+    /**
+     * @type {number}
+     * @protected
+     * @note No serialization as used as internal value that should not be changed by deserialization.
+     */
     this._nameId = gdjs.RuntimeObject.getNameIdentifier(this.name);
+    /** @type {string} */
     this.type = objectData.type || "";
+    /** @type {number} */
     this.x = 0;
+    /** @type {number} */
     this.y = 0;
+    /** @type {number} */
     this.angle = 0;
+    /** @type {number} */
     this.zOrder = 0;
+    /** @type {boolean} */
     this.hidden = false;
+    /** @type {string} */
     this.layer = "";
+    /** @type {boolean} */
     this.livingOnScene = true;
+    /**
+     * @type {number}
+     * @note No serialization as used as internal value that should not be changed by deserialization.
```

Comment:
Don't use @note at all, it's not supported by JSDoc

Reply:
@note is made for notes relevant to the code but not the documentation, so it's ignored by purpose by JSDoc

Reply:
Have you tried running JSDoc with these? From what I recall, @note is rejected as not recognised.

Reply:
No, but whatever, thoose notes are not relevant and removing them is not bad

Reply:
Yes that's why I think we should both remove them and not use @note at all.